simulation/halsim_ds_socket/src/main/native/cpp/main.cpp

```diff
@@ -124,33 +125,45 @@ static void SetupUdp(wpi::uv::Loop& loop) {
     });
   });
   simLoopTimer->Start(Timer::Time{100}, Timer::Time{100});
+  // DS Timeout
+  int timeoutMs = 100;
+  if (auto envTimeout = std::getenv("DS_TIMEOUT_MS")) {
+    try {
+      timeoutMs = std::stoi(envTimeout);
+    } catch (const std::exception& e) {
+      std::cerr << e.what() << '\n';
```

Comment:
```suggestion
      fmt::print(stderr, "Error parsing DS_TIMEOUT_MS: {}\n", e.what());
```

And include fmt/format.h instead of iostream.